torch/_dynamo/trace_rules.py

```diff
@@ -3317,12 +3317,12 @@ def check_file(filename, is_inlined_call=False):
     if any(filename.startswith(d) for d in get_legacy_mod_inlinelist()):
         return SkipResult(
             False,
-            "inlined according trace_rules.LEGACY_MOD_INLINELIST",
+            "LEGACY_MOD_INLINELIST",
```

Comment:
``LEGACY_MOD_INLINELIST`` will be appended with ``inlined according trace_rules.lookup {reasons.pop()}`` line 3410 and 3416